src/SFA.DAS.RoatpOversight.Web/Views/Shared/_GatewayOutcome.cshtml

```diff
@@ -60,7 +66,7 @@
             }
         </dd>
     </div>
-    @if (Model.GatewayReviewStatus == GatewayReviewStatus.Fail)
+    @if (Model.GatewayReviewStatus != GatewayReviewStatus.Pass)
```

Comment:
Is this intentional? I though the list was more than Pass/Fail/Removed ?

Reply:
Yes, the block shows if it's anything other than Pass